src/pl.x86.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "npp-pluginList",
-	"version": "1.5.9",
+	"version": "1.6.0",
```

Comment:
This version is just updated by the maintainer with N++ updates. Could you please undo this change.